src/logDownloaderService.py

```diff
@@ -0,0 +1,79 @@
+import os
+import time
+import hashlib
+import subprocess
+import psutil
+from git import Repo
+import shutil
+from loguru import logger
+
+LOGS_DIRECTORY = os.getenv('LOGS_DIRECTORY', '/path')
```

Comment:
Please use [`pathlib`](https://docs.python.org/3.14/library/pathlib.html), here and throughout the module, it's very convenient.
(It's also one of my favorite Python modules 🙃)